src/firefly/js/charts/ui/PlotlyToolbar.jsx

```diff
@@ -62,22 +62,33 @@ function getToolbarStates(chartId) {
     const hasSelection = !isEmpty(selection);
     const traceNames = data.map((t) => t.name).toString();
     const activeTraceType = get(data, `${activeTrace}.type`);
+    const selCount= getSelIndexesForTable(tbl_id,chartId).length;
+    const isScatter= isScatter2d(data?.[activeTrace]?.type);
     return {hasSelection, hasFilter, activeTrace, activeTraceType, tbl_id, hasSelected,
-            dragmode: get(layout, 'dragmode'), traceNames, columns};
+            dragmode: get(layout, 'dragmode'), traceNames, columns, selCount, isScatter};
 }
 
 
 function ScatterToolbar({chartId, expandable}) {
     const scatterToolbarState = useStoreConnector(() => getToolbarStates(chartId), [chartId]);
 
-    const {hasSelection, hasFilter, activeTrace, tbl_id, hasSelected, dragmode} = scatterToolbarState;
+    const {hasSelection, hasFilter, activeTrace, tbl_id, hasSelected, dragmode, selCount, isScatter} = scatterToolbarState;
+
+    useEffect(() => {
+        isScatter&&selCount>MAX_CHART_SELECT && showMaxSelectionWarningOnce();
+    }, [selCount]);
+
+
     const hasSelectionMode = Boolean(tbl_id);
     const help_id = getChartData(chartId)?.help_id;
 
     return (
         <Stack direction='row' alignItems='center'>
+            {isScatter && selCount>MAX_CHART_SELECT&&
+                <SelectionWarningBtn
+                    warningStr={`Scatter chart cannot show selection above ${MAX_CHART_SELECT}, currently there are ${selCount} points selected but not shown not the chart`}/>}
```

Comment:
This message needs some editing; discussing this on Slack.  I'll follow up with a specific recommendation here later.